rules.txt

```diff
@@ -384,6 +384,8 @@
 /homestead                                301  /services/property-lots-housing/get-real-estate-tax-relief/get-the-homestead-exemption/
 /homebuynow(.*)                           301  https://cedphilly.org/philadelphia-home-buy-now
 /humanrelations(.*)                       301  /departments/philadelphia-commission-on-human-relations/
+/ida-recovery                             301  /departments/office-of-the-director-of-finance/hurricane-ida-recovery-funding/
+/idarecovery                              301  /departments/office-of-the-director-of-finance/hurricane-ida-recovery-funding/
```

Comment:
Here instead of adding two different lines, you want to use a regex rule to make it one line. take a look at the business vaccine redirects on line 82. in order to have a character be optional in the redirect, you just need to add a ? after it. so for this one you just need ida-?recovery and you can delete the second line.